src/test/java/dk/kb/util/yaml/YAMLTest.java

```diff
@@ -94,6 +96,33 @@ public void testMultiConfig() throws IOException {
                      "The merged YAML should have the expected value for alias-using 'theServer'");
     }
 
+    @Test
+    public void testFailingMultiConfig() throws IOException {
+        try {
+            YAML.resolveMultiConfig("Not_there.yml", "Not_there_2.yml");
+            fail("Resolving non-existing multi-config should not work");
+        } catch (FileNotFoundException e) {
+            // Expected
+        }
```

Comment:
Perhaps utilising junit 5's expected exception may make it look cleaner?
See: https://howtodoinjava.com/junit5/expected-exception-example/

Reply:
I did not know that one. Very nice!

I have updated `YAMLTest` to take advantage of the assert.